fala/apps/adviser/tests/test_search_page_journeys_playwright.py

```diff
@@ -7,6 +7,23 @@ def test_landing_page(self):
         page = self.visit_search_page()
         expect(page.h1).to_have_text("Find a legal aid adviser or family mediator")
 
+    def test_landing_page_with_url_params(self):
+        page = self.visit_search_page_with_url_params("categories=hlpas")
+        expect(page.h1).to_have_text("Find a legal aid adviser or family mediator")
+        expect(page.error_list).not_to_be_visible()
+        expect(page.checkbox_by_label("Housing Loss Prevention Advice Service")).to_be_checked()
+
+    def test_landing_page_with_multiple_url_params(self):
+        page = self.visit_search_page_with_url_params("categories=hlpas&categories=edu")
+        expect(page.h1).to_have_text("Find a legal aid adviser or family mediator")
+        expect(page.error_list).not_to_be_visible()
+        expect(page.checkbox_by_label("Housing Loss Prevention Advice Service")).to_be_checked()
+        expect(page.checkbox_by_label("Education")).to_be_checked()
+        page._page.reload()
```

Comment:
Can we stick `Housing Loss Prevention Advice Service` into variables that we can use instead of having to type the same string out everytime.

Also a note as to why we refresh the page to perform the same checks.

Reply:
@psweeting1 have added variables and added comment